specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,38 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when a *capacity reservation* is unused.
+
+The CapacityReservationStatus column MUST be present in the billing data when the provider supports *capacity-reservations*. This column MUST be of type String, MUST be null when [CapacityReservationId](#capacityreservationid) is null, and MUST NOT be null when CapacityReservationId is not null and [Charge Category](#chargecategory) is "Usage". The CapacityReservationStatus MUST be one of the allowed values. The CapacityReservationStatus MUST label all unused *capacity reservation* charges and MUST label used *capacity reservation* charges if the provider supports it.
```

Comment:
```suggestion
The CapacityReservationStatus column adheres to the following requirements:

 * CapacityReservationStatus MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
 * CapacityReservationStatus MUST be null when CapacityReservationId is null and MUST NOT be null when CapacityReservationId is not null.
 * CapacityReservationStatus MUST be one of the allowed values.
```

Notes:

- The nullability of this column tied to CapacityReservationId, so any nullability-related updates (e.g, restriction to ChargeCategory Usage, etc.) performed on CapacityReservationId will be automatically reflected.
- Not sure if we should keep the labeling-related requirement, switch to explicitly specifying when a particular value MUST be used (see PricingCategory), or leave it as it is.

Reply:
Im fine with the formatting changes; Im not ready to commit to removing the labeling requirement.

Reply:
I'm okay with the labeling requirement as long as it doesn't require labeling used

Reply:
Attaching a subset of Pricing Category requirements as an example. 

> * PricingCategory MUST be one of the allowed values.
> * PricingCategory MUST be "Standard" when pricing is predetermined at the agreed upon rate for the [billing account](#glossary:billing-account).
> * PricingCategory MUST be "Committed" when [CommitmentDiscountId](#commitmentdiscountid) is not null.

I prefer this wording because it’s more precise and, after all, for consistency with the rest of the spec.

Reply:
see comment https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/523#discussion_r1738494230 for new suggestion with additional requirements